assignment/tool/src/webapp/vm/assignment/chef_assignments_list_assignments.vm

```diff
@@ -491,6 +496,28 @@
 									#end
 								#end
 							</td>
+							#if ($isStudent)
+								<td headers="grade">
+									#if ($service.getSubmissionStatus($submission.Id, false) == $tlang.getString("gen.returned"))
+										#if ($submission.grade.equals("pass"))
+											<span>$tlang.getString("pass")</span>
+										#elseif ($submission.grade.equals("fail"))
+											<span>$tlang.getString("fail")</span>
+										#elseif ($submission.grade.equals("Checked"))
+											<span>$tlang.getString("gen.checked")</span>
+										#elseif ($submission.grade.equals("Unchecked"))
+											<span>$tlang.getString("unchecked")</span>
+										#elseif ($assignment.TypeOfGrade.ordinal() == 3)
+											<span>$service.getFactoredGrade($assignment.ScaleFactor,$submission.grade)/$service.getMaxPointGradeDisplay($assignment.ScaleFactor, $assignment.MaxGradePoint)</span>
+										#else 
+											<span>$submission.grade</span>
+										#end
+									#else
+										<span>-#if ($assignment.TypeOfGrade.ordinal() == 3)/$service.getMaxPointGradeDisplay($assignment.ScaleFactor, $assignment.MaxGradePoint)#end</span>
```

Comment:
What is this '-' before the if?

Reply:
It's to get the -/100. When the teacher/instructor has not graded the assignments

Reply:
From the user perspective, '-' can mean other things, better using a text like 'Not graded' 'Sin calificar' 'Sense qualificar' ?

Reply:
Maybe string 'Not graded' could be confused... student doesn't know if the grade is really not graded or unpublished...
Lo que para el estudiante es "Sin calificar", para el docente puede que no lo sea ya que puede estar "Calificada" pero sin publicar ¿me explico?

Reply:
Cuando la nota de la tarea está sin publicar, si el estudiante pulsa en la tarea ve

![image](https://user-images.githubusercontent.com/15141743/215078600-8a6afe3f-d472-4eb3-85e2-011501aa7016.png)

Yo personalmente prefiero que aparezca algo que se vea que dicha tarea esta sobre 10. De ahí el -/10

Reply:
¿Aceptamos guión entonces? So do we accept the '-' symbol?

Reply:
Yes :)